### PR TITLE
feat: redirect user to welcome screen instead of closing the tab

### PR DESCRIPTION
**Pull request by raducristianpopa**
## Context

https://github.com/WICG/webmonetization/issues/461

## Changes proposed in this pull request

Instead of closing the tab we are now going to redirect the users to a welcome screen.

Additions:
- Error codes
- Interaction intent - `connect` or `funds` (to distinguish between user performing the setup and adding more funds)
- `OPEN_PAYMENTS_REDIRECT_URL` defined using Webpack: future proof, we might have a local version in the future if we want to have analytics on the welcome screen and we should avoid using the production one in dev mode.

### CODE REVIEW

**Comment by raducristianpopa** [comment]
<h2>Extension builds preview</h2>

<table role="table">
<thead>
  <tr>
    <th>Name</th>
    <th>Link</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td align="center">Latest commit</td>
    <td align="center">c4f2b3a17edca6fd7e51bfbcaa609575743315c2</td>
  </tr>
  <tr>
    <td align="center">Latest job logs</td>
    <td align="center"><a href="https://github.com/interledger/web-monetization-extension/actions/runs/10077348206">Run #10077348206</a></td>
  </tr>
  <tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/26374338162/artifacts/1735052058">Download</a></td></tr><tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/26374338162/artifacts/1735052062">Download</a></td></tr>
</tbody>
</table>

File: webpack/dev.ts [hunk 1]
```diff
@@ -44,7 +44,11 @@ export const getDevConfig = (target: Target): Configuration => {
         CONFIG_PERMISSION_HOSTS: JSON.stringify({
           origins: ['http://*/*', 'https://*/*']
         }),
-        CONFIG_ALLOWED_PROTOCOLS: JSON.stringify(['http:', 'https:'])
+        CONFIG_ALLOWED_PROTOCOLS: JSON.stringify(['http:', 'https:']),
+        CONFIG_OPEN_PAYMENTS_REDIRECT_URL: JSON.stringify(
```
Comment (raducristianpopa) [inline webpack/dev.ts:48]: Note to self: update this before merging.

**Review by raducristianpopa — COMMENTED** [review]
To be merged after https://github.com/WICG/webmonetization/issues/461 gets in.

**Review by sidvishnoi — APPROVED** [review]

**Review by dianafulga — APPROVED** [review]

File: webpack/dev.ts [hunk 2]
```diff
@@ -44,7 +44,10 @@ export const getDevConfig = (target: Target): Configuration => {
...
+          'https://webmonetization.org/welcome'
```
Comment (sidvishnoi) [inline webpack/dev.ts:49]: Should we stick to using `localhost:xyzz` one for dev?

Comment (raducristianpopa) [reply webpack/dev.ts:49]: It should be okay for now. We change to a local version in the future.

**Review by sidvishnoi — APPROVED** [review]

**Pull request merged**